doc/manual/source/physics/active_particles.rst

```diff
@@ -0,0 +1,137 @@
+    accretion process. If this is turned on then the ``SmartStarBHRadiativeFeedback`` should presumably be turned off unless you have a
+    good reason to include both. The efficiency, epsilon, depends on both the spin of the black hole and the ISCO oribit. In order to
+    calculate this accurately Eqn 32 from Abromowicz & Fragile (2013) is used. See ActiveParticle_SmartStar.h. The feedback is released
+    iostropically in a sphere surrounding the SmartStar particle.  Default: 0
+
+``SmartStarBHJetFeedback`` (external)
+    The methodology for this algorithm is based on that of Kim et. al (2011) (https://arxiv.org/pdf/1106.4007.pdf). Jets can be activiated
+    when a spinning black hole is accreting. The jets are bipolar and are set along the angular momentum vector of the SmartStar. The
+    velocity of the jet(s) is set by a separate parameter below. No other parameters need to be set to activate the jet.  Default: 0
+
+``SmartStarEddingtonCap`` (external)
+    This parameter allows for accretion onto the SmartStar to be capped at the Eddington limit. I see no good physical reason for doing this in
+    general.  Default: 0
+
+``SmartStarSpin`` (external)
+    The dimensionless spin of the SmartStar particle. This is a very uncontrained parameter and cannot be readily computed on the fly. This parameter
+    should be set if you want to have jet feedback. Setting this is zero and turning on jet feedback wouldn't make sense. The default is set to be 0.7 and
+    this is probably reasonable. 
+    Default: 0.7
+
+``SmartStarSMSLifetime`` (external)
+    This is the lifetime for a supermassive star. After this time has elapsed a SmartStar particle which is behaving like a SMS will collapase
+    directly into a black hole with no supernova event. Default: 1e6
+
+``SmartStarJetVelocity`` (external)
+    The velocity that the jets are ejected at. Typically jets are observed to travel at a substantial fraction of the speed of light -
+    especially those ejected during periods of high accretion. However,
+    as mass gets entrained on the jet it slows down. The units of this parameter are as a fraction of the speed of light. Default: 0.1
+
+``SmartStarFeedbackJetsThresholdMass`` (external)
+    Jets are only ejected once this amount of mass is available for ejected after an accretion event. Therefore, if there is very limited
+    accretion and this parameter is set high then jets will be very infrequent. In units of solar masses. Default: 1.0
+
+``SmartStarSuperEddingtonAdjustment`` (external)
+    As accretion rates exceed the canonical Eddington rate the radiative efficiency of the feedback changes. We use the fits from Madau et al. (https://arxiv.org/pdf/1402.6995.pdf)
+    to adjust the efficiency when accretion enters the super-critical regime. The fits are based on the slim-disk model of accretion which generate inefficient feedback.
+    Default: 1
+
```

Comment:
I think either this list of particle descriptions should either be copied in the parameter list, or moved there and linked from here?

Reply:
Hi DK, 
What did you mean here? The bit of text that is referred to here doesn't exactly match your comment.

Reply:
i meant this entire section lines 74-140.

Reply:
OK I see what you mean. I'll look into that. Thanks!

Reply:
Thanks DK that's done now.